tardis/conftest.py

```diff
@@ -56,6 +56,11 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
```

Comment:
Sure move it.

You can't rewrite history since part of the work is already merged.